velox/exec/prefixsort/benchmarks/PrefixSortAlgorithmBenchmark.cpp

```diff
@@ -0,0 +1,124 @@
+/*
+ * Copyright (c) Facebook, Inc. and its affiliates.
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+#include <folly/Benchmark.h>
+#include <folly/init/Init.h>
+#include "velox/buffer/Buffer.h"
+#include "velox/exec/prefixsort/PrefixSortAlgorithm.h"
+
+DEFINE_int32(sort_data_seed, 1, "random test data generate seed.");
+
+using namespace facebook::velox;
+using namespace facebook::velox::exec;
+
+namespace {
+namespace {
+// Encode logic will use PrefixSort`s encode module API as follow-up.
+#define BSWAP64(x) \
```

Comment:
This is a copy-paste from the test. I wonder if it makes sense to add basic support for encode/decode integers to production code in this PR and use that to avoid copy-pasting.